neuralprophet/metrics.py

```diff
@@ -4,10 +4,17 @@
 import pandas as pd
 import torch
 import logging
+import torchmetrics
 
 log = logging.getLogger("NP.metrics")
 
+METRICS = {
+    "MAE": torchmetrics.MeanAbsoluteError(),
+    "MSE": torchmetrics.MeanSquaredError(squared=True),
+    "RMSE": torchmetrics.MeanSquaredError(squared=False),
+}
```

Comment:
Info: We use torchmetrics to log metrics during training. Torchmetrics was originally part of the Lightning package but moved into a seperate package to reduce complexity. It is maintained by the same authors.